repos/re-theme/src/reStyle/reStyleHooks.js

```diff
@@ -28,6 +28,23 @@ export const getComponentName = Component => {
   )
 }
 
+/**
+ * Hook that ensures a consistent Object identity to avoid re-renders
+ * @function
+ * @param {Object} identity - Object to be reused
+ * @param {Array} mergeObjs - Other objects to be merged with the identity object
+ *
+ * @returns {Object} - identity object merged with the passed in mergeObjs
+ */
+export const useObjWithIdentity = (identity, ...mergeObjs) => {
```

Comment:
Is this still being used, or can we just remove it? Pretty sure it's only used in `reStyle`

Reply:
removed